src/game_interpreter.cpp

```diff
@@ -736,7 +736,7 @@ bool Game_Interpreter::CommandControlVariables(RPG::EventCommand const& com) { /
 			int a, b;
 			a = max(com.parameters[5], com.parameters[6]);
 			b = min(com.parameters[5], com.parameters[6]);
-			value = rand() % (a-b+1)+b;
+			value = Utils::GetRandomNumber(b,a);
```

Comment:
Interestingly the help of the official english version (both 2k/2k3) says the interval is exclusive ("For example, if the range “0~100” is specified, the value used will be a random number above 0 but below 100.").
RPG Advocates translation says "between 0 and 100" and Dons version doesn't explain the interval at all.

But I verified that the interval is indeed inclusive, code is correct.